csrc/ops/composite.h

```diff
@@ -57,6 +57,15 @@ TensorView* matmul(TensorView* a, TensorView* b);
 // can be used to create mamtuls without a cast operation following it.
 TensorView* matmul(TensorView* a, TensorView* b, bool cast_output_to_input);
 
+// Linear functions which takes in two tensors of shapes A[M,K] and
+// B[N,K]. Takes in a options bias of shape [N] and performs
+// out = A * B_Transpose + bias. The output dtype matches the dtype
+// ofthe inputs which should match.
+TensorView* linear(TensorView* a, TensorView* b, TensorView* bias);
+// This is an implemenation details to reflect when linear is called
```

Comment:
```suggestion
// This is an implemtenation details to reflect when linear is called
```
Also please explain why this is needed (is it?).